tests/System/integration/administrator/components/com_cache/Default.cy.js

```diff
@@ -0,0 +1,29 @@
+describe('Test in backend that the cache', () => {
+  beforeEach(() => {
+    cy.doAdministratorLogin();
+    cy.visit('/administrator/index.php?option=com_cache&view=cache');
+  });
+
+  it('have a title', () => {
+    cy.get('h1.page-title').should('contain.text', 'Maintenance: Clear Cache');
+  });
+
+  it('can display message', () => {
+    cy.get('div.alert.alert-info').should('contain.text', 'Select the Clear Expired Cache button');
+  });
+
+  it('can display a list of chached items', () => {
+    cy.get('tr.row0').should('contain.text', '_media_version');
+  });
+
+  it('can clear expired cache', () => {
+    cy.get('#toolbar-delete2').click();
+    cy.on('window:confirm', () => true);
```

Comment:
This test spec works in dev-4.4 for me and starting with 5.1-dev the following error occurs:
```
  1) Test in backend that the cache
       can clear expired cache:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Expired cached items have been cleared' within the element: <div#system-message-container> but never did.
      at Context.eval (webpack://joomla/./tests/System/integration/administrator/components/com_cache/Default.cy.js:22:91)
```
Reason is a new changed confirmation popup:

![warning](https://github.com/joomla/joomla-cms/assets/25269712/fbd8c2ee-5a4b-4d74-86c6-751b07c9c0ac)

I would recommend to make the test code release-independent to not maintain two different versions. I tested the following code change with all four branches successfully:

**Suggestion:**
```suggestion
    cy.on('window:confirm', () => true);
    // Starting with Joomla 5.1 check if the 'Yes' button exists and click it if present
    cy.get('body').then(($body) => {
      if ($body.find('div.buttons-holder button[data-button-ok]').length > 0) {
        cy.get('div.buttons-holder button[data-button-ok]').click();
      }
    });